File: src/main/resources/output/IFC4X3_java/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcControl/IfcWorkControl/IfcWorkControl.java
public class IfcWorkControl extends IfcControl {
    public IfcDateTime CreationDate;
    public SET [1:?] OF IfcPerson Creators;
    public IfcLabel Purpose;
    public IfcDuration Duration;
    public IfcDuration TotalFloat;
    public IfcDateTime StartTime;
    public IfcDateTime FinishTime;

    // === EXTENDED BY ===
    // IfcWorkPlan
    // IfcWorkSchedule
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcRoot/IfcObjectDefinition/IfcObject/IfcControl/IfcWorkControl/IfcWorkControl.py
class IfcWorkControl(IfcControl):
    def __init__(self):
        self.CreationDate: IfcDateTime = None
        self.Creators: SET [1:?] OF IfcPerson = None
        self.Purpose: IfcLabel = None
        self.Duration: IfcDuration = None
        self.TotalFloat: IfcDuration = None
        self.StartTime: IfcDateTime = None
        self.FinishTime: IfcDateTime = None

    # === EXTENDED BY ===
    # IfcWorkPlan
    # IfcWorkSchedule
